github/client.go

```diff
@@ -134,6 +134,23 @@ func (client *Client) CreatePullRequest(project *Project, params map[string]inte
 	return
 }
 
+func (client *Client) GetPullRequestUrlsForHead(headProject *Project, branch string) (pr []PullRequest, err error) {
```

Comment:
Could we have a generic `FetchPullRequests` method that accepts an arbitrary dictionary of query parameters, not just `head`, so we can use it for more purposes in the future? See `FetchIssues`.